live-examples/html-examples/text-content/main.html

```diff
@@ -0,0 +1,7 @@
+<header>
+    <img class='logo' src='/media/examples/lizard.png'>
```

Comment:
Nit: Code Style: Use double quotes in HTML. Also, explicitly close tags such as `img`. I know it is not required in HTML5 but, from a consistency perspective, I encourage that usage. Thanks @a2sheppy